runners/katacoda/index.ts

```diff
@@ -464,6 +468,13 @@ export class Katacoda extends Runner {
     }
 
     private renderTemplate(name: string, targetPath: string, variables) {
+        if("text" in variables) {
+            variables["text"] = this.converter.convertAsciidocToMarkdown(variables["text"]);
+        }
+        if("textAfter" in variables) {
+            variables["textAfter"] = this.converter.convertAsciidocToMarkdown(variables["textAfter"]);
+        }
```

Comment:
Maybe also check for playbook.description and convert it to markdown. Now we don´t have a problem with that but somebody could also use asciidoc formating in the description in the future.

Reply:
If I implemented it correctly, the description conversion can be found [here](https://github.com/devonfw-tutorials/tutorial-compiler/pull/288/files#diff-720eb155dc2cef159b53febe6c14ddfdd780d241514c2f056eb71a932c4146c9R69)